frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -367,9 +367,9 @@
             // ev.stopPropagation();
             var participationState;
             if (isRegistrationOpen) {
-                participationState = 'Close';
+                participationState = 'closed';
             } else {
-                participationState = 'Open';
+                participationState = 'opened';
```

Comment:
@deshraj If you change this(^) text here then you also need to take care of the text which is shown over the confirmation modal [here](https://github.com/Cloud-CV/EvalAI/pull/2802/files#diff-14a845646ed48757b5c7798b7ccdddb6R375). Cause it will be incorrect to ask `opened/closed participation in the challenge?`

Reply:
Good point. Totally missed it. Incorporating now. Thanks!